src/js/upcoming-this-month.js

```diff
@@ -0,0 +1,97 @@
+import TmdbApi from './tmdb-api.js';
+
+const tmdb = new TmdbApi();
+
+async function getUpcomingMovies() {
+  try {
+    const movies = await tmdb.getUpcomingMovies();
+    const today = new Date();
+    const thisMonthMovies = movies.filter(movie => {
+      const releaseDate = new Date(movie.release_date);
+      return (
+        releaseDate.getFullYear() === today.getFullYear() &&
+        releaseDate.getMonth() === today.getMonth()
+      );
+    });
+
+    if (thisMonthMovies.length === 0) {
+      displayMessage('No upcoming movies this month.');
+    } else {
+      const randomMovie =
+        thisMonthMovies[Math.floor(Math.random() * thisMonthMovies.length)];
+      displayMovie(randomMovie);
+    }
+  } catch (error) {
+    console.error('Failed to fetch upcoming movies:', error);
```

Comment:
tutaj możesz dodać displayMessage() z wyświetleniem info dla użytkownika
bo teraz to jest error tylko w konsoli

Reply:
ok, dodane